documentation/staging/content/userguide/base-images/access-images.md

```diff
@@ -0,0 +1,92 @@
+and set up image pull secrets for a domain resource:
+
+- A Kubernetes `docker-registry` secret containing the registry credentials must be created
+  in the same namespace as domain resources with a `domain.spec.image` attribute that reference the image.
+  For example, to create a secret with OCR credentials:
+
+  ```shell
+  $ kubectl create secret docker-registry SECRET_NAME \
+    -n NAMESPACE_WHERE_YOU_DEPLOY_DOMAINS \
+    --docker-server=container-registry.oracle.com \
+    --docker-username=YOUR_USERNAME \
+    --docker-password=YOUR_PASSWORD \
+    --docker-email=YOUR_EMAIL
+  ```
+
+- The name of the secret must be added to these domain resources using
+  the `domain.spec.imagePullSecrets` field. For example:
+
+  ```text
+  ...
+  spec:
+  ...
+    imagePullSecrets:
+    - name: SECRET_NAME
+  ...
+  ```
+
+- If you are using the Oracle Container Registry, then
+  you must use the web interface to accept the Oracle Standard Terms and Restrictions
+  for the Oracle software images that you intend to deploy.
+  You need to do this only once for a particular image.
+  See [Obtain images from the Oracle Container Registry](#obtain-images-from-the-oracle-container-registry).
+
+For more information about creating Kubernetes Secrets for accessing
+the registry, see the Kubernetes documentation,
+[Pull an image from a private registry](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/).
+
+#### Option 2: Store images in a central registry and set up a Kubernetes service account with image pull secrets in each domain namespace
+
+An option for accessing an image that is stored in a private registry
+is to set up the Kubernetes `ServiceAccount` in the namespace running the
+WebLogic domain with a set of image pull secrets thus avoiding the need to
+set `imagePullSecrets` for each `Domain` resource being created (because each resource
+instance represents a WebLogic domain that the operator is managing):
+
+- Create a Kubernetes `docker-registry` secret as shown
+  in [Option 1](#option-1-store-images-in-a-central-registry-and-set-up-image-pull-secrets-on-each-domain-resource).
+
+- Modify the `ServiceAccount` that is in the same namespace
+  as your domain resources to include this image pull secret:
+
+  ```shell
+  $ kubectl patch serviceaccount default -n domain1-ns \
+  -p '{"imagePullSecrets": [{"name": "my-registry-pull-secret"}]}'
+  ```
+
+  Note that this patch command entirely replaces the current list of
+  image pull secrets (if any). To include multiple secrets, use
+  the following format:
+  `-p '{"imagePullSecrets": [{"name": "my-registry-pull-secret"}, {"name": "my-registry-pull-secret2"}]}'`.
+
+For more information about updating a Kubernetes `ServiceAccount`
+for accessing the registry, see the Kubernetes documentation,
+[Configure service accounts for pods](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#add-image-pull-secrets-to-a-service-account).
```

Comment:
This should be a relref - not a full link

Reply:
Not sure why a link to the Kubernetes docs needs to be a relref and I cannot figure out how to do it. So, for now, I'm leaving it as is.

Reply:
Oops - sorry - didn't look closely - I mistakenly thought this was a ref into our docs. Resolving.